File: JAVA/string_rev.java
import java.util.Scanner;

public class string_rev {

    public static String reverseString(String text) {
        String reverse = "";

        for(int i = text.length() - 1; i >= 0; i--) {
            reverse += text.charAt(i);
        }

        return reverse;
    }

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a string: ");
        String word = sc.nextLine();

        System.out.println(reverseString(word));
    }
}

File: PY/string_rev.py
def reverse_string(text):
    return text[::-1]

word = input("Enter a string: ")
print(reverse_string(word))

#without slicing

def reverse_string(text):
    reverse = ""
    for ch in text:
        reverse = ch + reverse
    return reverse

word = input("Enter a string: ")
print(reverse_string(word))